packages/web/src/components/lineup/TanQueryLineup.tsx

```diff
@@ -40,7 +48,24 @@ const { makeGetCurrent } = queueSelectors
 
 export interface TanQueryLineupProps {
   /** Query data should be fetched one component above and passed through here */
-  lineupQueryData: LineupQueryData
+  data:
+    | (
+        | UserTrackMetadata
```

Comment:
Surprised this didn't get generic-d, but I suppose that's a lot more work below with customizing the types of other props/fields based on the incoming data type.